message/router.go

```diff
@@ -34,6 +34,11 @@ type HandlerFunc func(msg *Message) ([]*Message, error)
 // NoPublishHandlerFunc is HandlerFunc alternative, which doesn't produce any messages.
 type NoPublishHandlerFunc func(msg *Message) error
 
+// ProxyHandler is a handler that passes the message unchanged from the subscriber to the publisher.
+var ProxyHandler HandlerFunc = func(msg *Message) ([]*Message, error) {
```

Comment:
maybe `PassthroughHandler` would better show what it is doing? probably @maclav3 can add something :P

Reply:
666e8e97d7b8637930ea7140a41fe12ed9b6bea4

Reply:
I was thinking about `NoopHandler` (not perfect, because it's not only noop, but also returns the incoming message) or `IdentityHandler` (probably too obscure and mathematical).
I thought about `TransparentHandler` or `TransmitHandler`, but it's also kinda meh

Reply:
I like the proposed "passthrough", as the behavior is close to the logic gate with the same name. `NoopHandler` would look more like:

```go
func(msg *Message) ([]*Message, error) {
    return nil, nil
}
```

I even thought about adding it as well, but then again I can't think of any use for such handler.